GitUI/CommandsDialogs/SettingsDialog/MergeToolsHelper.cs

```diff
@@ -74,6 +74,21 @@ public static string FindFileInFolders(string fileName, params string[] location
                         }
                     }
                 }
+
+                string localAppDataPath = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
```

Comment:
Could you explain what this section is for please?

Reply:
Of course. 

As I explained in the commit message, Atom is installed (silently and unwillingly, haha) in %LocalAppData%, I had to expand `FindFileInFolders` method to search there as well.

Reply:
Done.

Reply:
Thanks. I'm wondering if it's correct though, as the first caller to `CheckFileExists` could return false on the `File.Exists` line, at which point `fileName` has been reassigned to a full path. Subsequent calls to `CheckFileExists` then end up having a full file path which they try to combine with another location.

Reply:
I wondered about this too, but it worked.
I fixed it just in case, better be safe than sorry.

Reply:
That looks more sensible 👍